sources/tests/Unit/Command/InspectSchema.php

```diff
@@ -37,6 +37,9 @@ public function testExecute()
                 'command'          => $command->getName(),
                 'config-name'      => 'pomm_test',
                 'schema'           => 'pomm_test',
+            ],
+            [
+                'decorated' => false
```

Comment:
Could not find in Sf sources what this option actually does. Still, it is a good idea to set it in case default value changes.